R/simulate.r

```diff
@@ -217,11 +237,22 @@ simulate_tree <- function(nchains, statistic = c("size", "length"),
 
 
 
-#' Simulate a summary of the transmission chain statistic
+#' Simulate transmission chains sizes/lengths without infection tree
 #'
 #' @inheritParams simulate_tree
 #' @param stat_max A cut off for the chain statistic (size/length) being
 #' computed. Results above the specified value, are set to `Inf`.
+#' @inheritSection simulate_tree Calculating chain sizes and lengths
+#' @inheritSection simulate_tree The serial interval (`serials_dist`)
+#' @author James M. Azam, Sebastian Funk
+#' @seealso [simulate_tree()] for simulating the transmission chains,
+#' returning the full tree information ("sim_id", "chain_id",
+#' "ancestor", "generation", and optionally, "time").
```

Comment:
```suggestion
#' @seealso [simulate_tree()] for simulating transmission trees,
#' i.e. retaining information on individual transmission events.
#' (but otherwise using the exact same model as here).
```